resources/views/layouts/app.blade.php

```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <title>Laravel</title>
+    <title>Laravel @yield('page-title')</title>
```

Comment:
Could you add a default value?

Reply:
ok no problem but you are a terrbile boss